application/frontend/controllers/MfaController.php

```diff
@@ -112,41 +111,45 @@ public function actionDelete($mfaId)
 
     /**
      * @param $mfaId
-     * @throws BadRequestHttpException
-     * @throws NotFoundHttpException
-     * @throws ServiceException
-     * @throws TooManyRequestsHttpException
+     * @return array|bool
+     * @throws HttpException
      */
     public function actionVerify($mfaId)
     {
+        $messages = [
+            400 => 'Invalid code provided',
+            404 => 'MFA verify failure',
+            429 => 'MFA rate limit failure',
+        ];
+
+        $codes = [
```

Comment:
Maybe this way of doing things was agreed on in the past, but if not ...

I'm wondering if this may not be helpful in the end.  My understanding was that error codes as timestamps were to help us programmers find the line where the error was thrown.  

I'm thinking the message will be enough to tell us what kind of error it was, and then we can just use a single code in the `catch` block.

Reply:
Good call, thanks.